catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -236,15 +260,17 @@
 
     protected Converter cswTransformConverter;
 
+    protected MessageBodyWriter<CswTransactionRequest> cswTransactionWriter;
+
     protected String forceSpatialFilter = NO_FORCE_SPATIAL_FILTER;
 
     protected ScheduledFuture<?> availabilityPollFuture;
 
     protected SecurityManager securityManager;
 
-    private FilterBuilder filterBuilder;
+    protected FilterBuilder filterBuilder;
```

Comment:
For an old version of registry store i needed it. I have reverted these changes.